### PR TITLE
Use data folder for packages

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

**Comment by clementbiron** [comment]
I have a little doubt, should we use `webAPI` or `endpoint` as metadata name?

File: data/packages/aotearoa-new-zealand.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title:
+  fr: Aotearoa Nouvelle-Zélande
+  en: Aotearoa New Zealand
+jurisdiction: NZ
+website: http://nz.openfisca.org/
```
Comment (MattiSG) [inline data/packages/aotearoa-new-zealand.yml:5]:
```suggestion
website: https://nz.openfisca.org/
```

File: data/packages/france.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+title:
+  fr: France
+  en: France
+jurisdiction: FR
+website: https://fr.openfisca.org/legislation
+source: https://github.com/lesmevesajudes/openfisca-barcelona
```
Comment (MattiSG) [inline data/packages/france.yml:6]: 😉 
```suggestion
source: https://github.com/openfisca/openfisca-france
```

File: data/packages/tunisia.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+title:
+  fr: Tunisie
+  en: Tunisa
+jurisdiction: TN
+website: https://www.openfisca.tn
```
Comment (MattiSG) [inline data/packages/tunisia.yml:5]: Their certificate is broken (poke @sandcha).
```suggestion
website: http://www.openfisca.tn
```

File: data/packages/united-states.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+title:
+  fr: États-Unis
+  en: United States
```
Comment (MattiSG) [inline data/packages/united-states.yml:3]:
```suggestion
  en: United States of America
```

File: data/packages/country-template.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+title:
+  fr: Modèle de pays
+  en: Country Template
+jurisdiction: ZZ #ISO 3166-2 format
+website: https://openfisca.org
+source: https://github.com/openfisca/country-template
+webAPI: https://legislation.fr.openfisca.org
```
Comment (MattiSG) [inline data/packages/country-template.yml:7]: 😉 
```suggestion
webAPI: https://legislation.demo.openfisca.org
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
I love it, this is a beautiful as hoped for 😍 

There is however one problem: inconsistent usage of `webAPI`, sometimes targeting the Legislation Explorer, sometimes the interactive OpenAPI page (“swagger”), sometimes the endpoint itself. I guess that proves the point that the name is ambiguous 😄

What about the following metadata structure to replace `webAPI`:

```yaml
legislation_explorer: # URL to an interactive user interface for navigating implemented legislation, such as https://legislation.demo.openfisca.org or https://nz.openfisca.org
openapi_spec: # URL to an OpenAPI specification file, such as https://api.demo.openfisca.org/latest/spec
```

For the case of NZ, `website` and `legislation_explorer` would be the same, for example 🙂

**Comment by MattiSG** [comment]
I see how in the future we could add more metadata such as `package_name` to give custom pypy installation instructions or `endpoint` to automatically expose some data, but until we build a user interface for it, let's keep things lean. The OpenAPI spec is the most generic thing we can get, from there we can auto discover the endpoint 🙂

**Comment by MattiSG** [comment]
Netlify builds fail because the base image is outdated. Can you handle that correction @clementbiron? 🙂 
@sandcha I have no knowledge about this process, if necessary, can you assist?

**Comment by MattiSG** [comment]
Also, please make sure that the URLs are updated or removed if broken so we can close https://github.com/openfisca/openfisca.org/issues/91 🙂

**Comment by clementbiron** [comment]
Good idea to replace `webAPI` by `legislation_explorer` and `openapi_spec`! I did a commit to update that.

**Review by MattiSG — APPROVED** [review]
🏗️ 🚀

**Pull request merged**